.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discussions
+    url: https://github.com/OrchardCMS/OrchardCore/discussions/new
+    about: Is your issue perhaps more of a discussion? Please open topics for discussion using the new GitHub Discussions feature.
```

Comment:
Perhaps add this as a comment to both templates along the lines of...?

```
<!-- Do you have questions about Orchard rather than wanting to change something in Orchard? Then your issue is perhaps more of a discussion. Please open topics for discussion using the new GitHub Discussions feature: https://github.com/OrchardCMS/OrchardCore/discussions/new. -->
```

Reply:
They will be suggested to select the Discussions link during the first step, so I don't think it is needed to recall this in the other templates.

Reply:
Ah, OK then.